app/main.py

```diff
@@ -1,34 +1,83 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self,
+                 row: int,
+                 column: int,
+                 is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(self,
+                 start: tuple[int],
+                 end: tuple[int],
+                 is_drowned: bool = False) -> None:
+        if not (start[0] == end[0] or start[1] == end[1]):
+            raise ValueError("Ships shouldn't be located "
```

Comment:
This error message doesn't correspond with the check performed. It will only be raised when is not one-dimensional (e.g. its width is not 1).